exwm-xsettings.el

```diff
@@ -181,6 +181,21 @@ These settings take precedence over `exwm-xsettings-theme' and
    (owner :initarg :owner :type xcb:WINDOW))      ;new slot
   :documentation "An XSETTINGS client message.")
 
+(eval-and-compile
+  (fset 'exwm-xsettings--color-dark-p
+        (if (< emacs-major-version 29)
```

Comment:
I suggest to write this in a slightly different form with `defalias`, which I consider cleaner:

~~~ elisp
(defalias 'exwm-xsettings--color-dark-p 
   (if (eval-when-compile (< emacs-major-version 29)) ...))
~~~

Reply:
Fixed and amended.